x-pack/plugins/security_solution/public/app/home/global_header/index.tsx

```diff
@@ -83,12 +82,6 @@ export const GlobalHeader = React.memo(
       };
     }, [portalNode, setHeaderActionMenu, theme.theme$]);
 
-    const { isTourShown, endTour } = useTourContext();
```

Comment:
not sure why this was here, but we are changing the tour entirely so i think safe to remove?